### PR TITLE
Add @lbernick, @betamos, @bIgBV to wg-async-foundations

### PR DESCRIPTION
**Pull request by tmandry**
And sort the list of members.

### CODE REVIEW

**Comment by tmandry** [comment]
cc @niko

**Comment by tmandry** [comment]
cc @nikomatsakis

File: people/bIgBV.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name = 'Bhargav Voleti'
+github = 'bIgBV'
+github-id = 5019938
+zulip-id = 248777
```
Comment (bIgBV) [inline people/bIgBV.toml:4]:
```suggestion
zulip-id = 248777
email = 'bhargav.voleti93@gmail.com'
```

**Comment by tmandry** [comment]
Ready to merge.

**Pull request merged**